packages/smooth_app/lib/generic_lib/buttons/smooth_large_button_with_icon.dart

```diff
@@ -32,13 +34,19 @@ class SmoothLargeButtonWithIcon extends StatelessWidget {
                 : Colors.blue,
           ),
           const Spacer(),
-          Text(
-            text,
-            textAlign: TextAlign.center,
-            style: themeData.textTheme.bodyText2!.copyWith(
-              color: isDarkMode
-                  ? Theme.of(context).colorScheme.onPrimary
-                  : Colors.blue,
+          ConstrainedBox(
+            constraints: BoxConstraints(
+              maxWidth: size.width * 0.5,
+            ),
+            child: AutoSizeText(
+              text,
+              maxLines: 2,
+              // textAlign: TextAlign.center,
```

Comment:
Why did you remove this line?
If that's irrelevant now, please delete it completely

Reply:
Sure.